lib/schema/loop.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2015, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+'use strict';
+
+module.exports = (function () {
+  // NOTE: These are static values rather than environment variables to make sure they cannot be easily changed
+  // without review and approval.
+  return {
+    minimumVersion: "0.1.0",
```

Comment:
I checked this with Rick Pasetto, let's make this "1.1.0" as that is the earliest version of Tidepool Loop that would ever launch.